components/TextFields/src/TypographyThemer/MDCTextFieldTypographyThemer.h

```diff
@@ -22,10 +22,8 @@
  details on replacement APIs.
  Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
  */
-@interface MDCTextFieldTypographyThemer : NSObject
-@end
-
-@interface MDCTextFieldTypographyThemer (ToBeDeprecated)
+__deprecated_msg("Please use the Theming extension instead.")
```

Comment:
Please point to the method they will need to use or build target/podspec. I worry that someone without context wouldn't know what a theming extension is.